test/unit/models/users.test.js

```diff
@@ -117,6 +117,24 @@ describe("users", function () {
     });
   });
 
+  describe("fetch user details based on discord id", function () {
```

Comment:
here expect(result.user).to.haveOwnProperty("state");. The result.user object is the result of the users.fetchUser() function, which returns a user object. The state property is not part of the user object, so the expect() assertion will fail.

To fix this, you need to remove the expect(result.user).to.haveOwnProperty("state"); assertion.

Reply:
![image](https://github.com/Real-Dev-Squad/website-backend/assets/53934353/08b50c70-8ed2-4e06-a606-4dc9a8c45c9e)
I am adding the state property in the beforeEach block.

Reply:
This is the updated user object after adding the state property in the beforeEach block. 
![image](https://github.com/Real-Dev-Squad/website-backend/assets/53934353/74b64be1-c7c2-4bda-b01a-c3510ff23914)